scripts/linters/js_ts_linter.py

```diff
@@ -56,6 +56,20 @@
 _MESSAGE_TYPE_SUCCESS = 'SUCCESS'
 _MESSAGE_TYPE_FAILED = 'FAILED'
 
+_BANNED_ESLINT_DISABLE_STATEMENTS = [
+    {
+        'pattern': r'eslint-(disable|enable)(-next-line)? camelcase',
```

Comment:
Can we use `BAD_PATTERNS_JS_AND_TS_REGEXP` instead of creating a new block here?

Reply:
done! didn't know we had something like this.